relay/sources/rudder-pkg/rust/rudder-package/src/main.rs

```diff
@@ -0,0 +1,76 @@
+use std::{
+    collections::HashMap,
+    fs,
+};
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize)]
+struct PluginMetadata {
+    #[serde(rename(serialize = "type", deserialize = "type"))]
```

Comment:
```suggestion
    #[serde(rename = "type")]
```

and the same for others